### DISCUSSION TITLE
input type file validation error

### DISCUSSION THREAD

**Discussion by csantos-nydig** [capricorn86/happy-dom#1495] [Q&A]
I'm mostly curious why this validation exists:

https://github.com/capricorn86/happy-dom/blob/e6f91276d7b4549d17f5d67bd2f909723f4940b3/packages/happy-dom/src/nodes/html-input-element/HTMLInputElement.ts#L795-L802

And if it can be removed or updated?

### Context

I'm using `react-dropzone` and they have this line https://github.com/react-dropzone/react-dropzone/blob/c36ab5bd8b8fd74e2074290d80e3ecb93d26b014/src/index.js#L807-L808
which triggers and error in `vitest` + `happy-dom`

### Plain JS example

https://codesandbox.io/p/sandbox/silly-kowalevski-qjrmt6

```
const fileInput = document.getElementById("my-file");
fileInput.value = null;
```
doesn't throw errors in any browser

**Comment by capricorn86** [comment]
I will close this as it was fixed in #1604
